### ISSUE TITLE
user: "perfsonar-toolkit-servicewatcher" not "perfsonar-toolkit-service-watcher" 

### ISSUE THREAD

**Issue by igarny** [perfsonar/docs#40]
http://docs.perfsonar.net/release_candidates/4.0rc1/install_centos.html 
Also under Step 2, its "perfsonar-toolkit-servicewatcher" not "perfsonar-toolkit-service-watcher" and the optional packages python install script simply doesnt exist at all as best as I can tell.

```
From: perfsonar-user-request@internet2.edu [mailto:perfsonar-user-request@internet2.edu] On Behalf Of Daniel Spisak
Sent: Mittwoch, 15. Februar 2017 04:39
To: perfsonar-user@internet2.edu
Subject: [perfsonar-user] Questions about perfsonar 4

```

**Comment by igarny** [comment]
correct link for reference is: http://docs.perfsonar.net/release_candidates/4.0rc1/install_centos.html#step-2-install-rpm

**Comment by szymontrocha** [comment]
RC2 link: http://docs.perfsonar.net/release_candidates/4.0rc2/install_centos.html#step-2-install-rpm

**Comment by tonin** [comment]
Corrected by 5b91e451b1b840400ce85aadf801ef411e9d32bd
